pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -354,6 +368,11 @@ func (v Vertex) GetPodSpec(req GetVertexPodSpecReq) (*corev1.PodSpec, error) {
 		MountPath: RuntimeDirMountPath,
 	})
 
+	sidecarContainers[0].VolumeMounts = append(sidecarContainers[0].VolumeMounts, corev1.VolumeMount{
```

Comment:
Though about it again, doing it in each different vertex type might be better.

Also, to be consistent, add the volume mount to numa container as well.